hack/istio/playbooks/README.adoc

```diff
@@ -48,13 +38,22 @@ You must choose between Ingress Route or Openshift Route there is a traffic gene
 - If you want to use ingress route (all traffic is going to hit ingress-gateway), you may use the following command:
 
 ```
-ansible-playbook bookinfo.yml -e istio_version=1.0.0 -e ingress_route=true -e traffic_generator=true -e rate=1 -vv
+ansible-playbook bookinfo.yml -e ingress_route=true -e traffic_generator=true -e rate=1 -vv
 ```
 
 === Kiali Traffic Generator with Openshift Route
 
 - If you want to use openshift route (kiali will report as "unknown"), you may use the following command:
 
 ```
-ansible-playbook bookinfo.yml -e istio_version=1.0.0 -e ingress_route=false -e traffic_generator=true -e rate=1 -vv
+ansible-playbook bookinfo.yml -e ingress_route=false -e traffic_generator=true -e rate=1 -vv
+```
+
+
+=== Kiali Traffic Generator with MongoDB
```

Comment:
Since this also supports MySQL, you should add a section about that, too ("Kiali Traffic Generator with MySQL DB").